Testing any contract including the default greeter no longer works, ABI is None

Hi, great library, I have been using it for a while now, but recently it stopped working (maybe I updated something?).

* Populus Version: 2.0.2
* OS: Linux (Ubuntu 16.04)
* Python web3 Version:  3.16.4
* solc Version: 0.4.19+commit.c4cbbb05.Linux.g++

### What was wrong?

I simply ran `populus init` and `popolus compile` followed by `pytest tests` and
this fails immediately with:

```

    def test_custom_greeting(chain):
>       greeter, _ = chain.provider.get_or_deploy_contract('Greeter')

tests/test_greeter.py:9: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
../../anaconda3/envs/trump/lib/python3.6/site-packages/populus/contracts/provider.py:143: in get_or_deploy_contract
    deploy_kwargs=deploy_kwargs,
../../anaconda3/envs/trump/lib/python3.6/site-packages/populus/contracts/provider.py:121: in deploy_contract
    kwargs=deploy_kwargs,
../../anaconda3/envs/trump/lib/python3.6/site-packages/web3/contract.py:308: in deploy
    deploy_transaction['data'] = cls._encode_constructor_data(args, kwargs)
../../anaconda3/envs/trump/lib/python3.6/site-packages/eth_utils/string.py:85: in inner
    return force_obj_to_text(fn(*args, **kwargs))
../../anaconda3/envs/trump/lib/python3.6/site-packages/web3/contract.py:741: in _encode_constructor_data
    constructor_abi = get_constructor_abi(cls.abi)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

contract_abi = None

    def get_constructor_abi(contract_abi):
        candidates = [
>           abi for abi in contract_abi if abi['type'] == 'constructor'
        ]
E       TypeError: 'NoneType' object is not iterable

../../anaconda3/envs/trump/lib/python3.6/site-packages/web3/utils/abi.py:230: TypeError
=========================================================== 2 failed in 0.49 seconds ==========================================================
```

#### Cute Animal Picture

![Kitty](https://i1.wp.com/catnamescity.com/wp-content/uploads/2012/08/Cute-Names-for-Cats.jpg?w=233)

--- a/populus/compilation/backends/solc_standard_json.py
+++ b/populus/compilation/backends/solc_standard_json.py
@@ -117,11 +117,11 @@ class SolcStandardJSONBackend(BaseCompilerBackend):
                 'remappings': import_remappings,
                 'outputSelection': {
                     '*': {
-                        '*': ['metadata',
-                              'evm.bytecode',
-                              'evm.bytecode.linkReferences',
-                              'evm.deployedBytecode',
-                              'evm.deployedBytecode.linkReferences']
+                        '*': [
+                            'abi',
+                            'metadata',
+                            'evm',
+                        ]
                     }
                 }
             }

